CONTRIBUTING.md

```diff
@@ -1,54 +1,124 @@
-development helper commands, building, testing, container images and local
-cluster.
 
 # Open a Pull Request
-Kubernetes-nmstate generally follows the [standard github pull request process](https://gist.github.com/Chaser324/ce0505fbed06b947d962), but there is a layer of additional specific differences:
 
-The first difference you'll see is that a bot will begin applying structured labels to your PR.
+kubernetes-nmstate generally follows the [standard github pull request
+process](https://gist.github.com/Chaser324/ce0505fbed06b947d962), but there is a
+layer of additional specific differences:
 
-The bot may also make some helpful suggestions for commands to run in your PR to facilitate review.
-These /command options can be entered in comments to trigger auto-labeling and notifications. Refer to its command reference documentation.
+The first difference you'll see is that a bot will begin applying structured
+labels to your PR.
+
+The bot may also make some helpful suggestions for commands to run in your PR to
+facilitate review. These `/command` options can be entered in comments to
+trigger auto-labeling and notifications. Refer to its command reference
+documentation.
 
 Common new contributor PR issues are:
 
-Missing DCO sign-off:
-    Developers Certificate of Origin (DCO) Sign-off is a requirement for getting patches into the project (see [Developers Certificate of Origin](https://developercertificate.org/)).
-    You can "sign" this certificate by including a line in the git commit of "Signed-off-by: Legal Name <email-address>".
-    If you forgot to add the sign-off, you can also amend your commit with the sign-off:
-    `git commit --amend -s`.
+- Missing DCO sign-off:\
+  Developers Certificate of Origin (DCO) Sign-off is a requirement for getting
+  patches into the project (see [Developers Certificate of
+  Origin](https://developercertificate.org/)). You can "sign" this certificate
+  by including a line in the git commit of "Signed-off-by: Legal Name
+  <email-address>". If you forgot to add the sign-off, you can also amend your
+  commit with the sign-off: `git commit --amend -s`.
 
 ## Code Review
 
-To make it easier for your PR to receive reviews, consider the reviewers will need you to:
+To make it easier for your PR to receive reviews, consider the reviewers will
+need you to:
 
-- Follow the project [coding conventions](https://github.com/golang/go/wiki/CodeReviewComments).
+- Follow the project [coding
+  conventions](https://github.com/golang/go/wiki/CodeReviewComments).
 - Write [good commit messages](https://chris.beams.io/posts/git-commit/).
-- Break large changes into a logical series of smaller patches which individually make easily understandable changes, and in aggregate solve a broader issue.
+- Break large changes into a logical series of smaller patches which
+  individually make easily understandable changes, and in aggregate solve a
+  broader issue.
```

Comment:
Also state that we squash PRs, and that multi commit PRs are not that helpful.

Reply:
Ditto

It is helpful for reviews though